src/ocm/components/clusterDetail/ProgressBarAlerts.tsx

```diff
@@ -38,6 +39,37 @@ const getFailedOperatorsNames = (failedOperators: MonitoredOperator[], t: TFunct
   return failedOperatorsNames;
 };
 
+const getFailedHostsMessage = (
+  failedHosts: number,
+  totalHosts: number,
+  message?: string,
+  hostsType?: string,
+) => {
+  if (hostsType === 'worker') {
+    if (ocmClient) {
+      return (
+        <>
+          Error information for each host can be found in the host inventory table. To retry adding
+          failed hosts, navigate to the clusters list inside OpenShift Cluster Manager, click your
+          cluster name, and select the <b>Add hosts</b> tab.
+        </>
+      );
+    }
+    return (
+      <>
+        Error information for each host can be found in the host inventory table. To retry adding
+        failed hosts, click the <b>Add hosts</b> button below.
```

Comment:
```suggestion
        failed hosts, press the <b>Add hosts</b> button.
```

Reply:
Nit text change in order to not commit to a need for having a mouse in order to activate the button and removing below in case we decide to reposition the button somewhere else in the screen